lib/rubocop/cop/rspec/stubbed_mock.rb

```diff
@@ -157,18 +162,7 @@ def on_expectation(expectation, method_name, matcher)
         def msg(method_name)
           format(MSG,
                  method_name: method_name,
-                 replacement: replacement(method_name))
-        end
-
-        def replacement(method_name)
-          case method_name
-          when :expect
-            :allow
-          when :is_expected
-            'allow(subject)'
-          when :expect_any_instance_of
-            :allow_any_instance_of
-          end
+                 replacement: REPLACEMENTS[method_name])
```

Comment:
Simplecov would complain if we used `#fetch` here, right?

Reply:
Simplecov is fine with using `fetch` -- it's slightly less performant than simple lookup. 

And without a fallback value it raises an error, which I think we shouldn't do without error handling. 

Not sure how to proceed here.